.github/workflows/build.yml

```diff
@@ -37,13 +37,17 @@ jobs:
       - name: Build
         run: |
           make clean
-          make build-linux
-          make package-linux
+          make build
+          make package
```

Comment:
Maybe it's better to `build` after both `test-linux` and `test-windows` finish successfully.
@ekcasey, what do you think?

Reply:
sure thing!